css/bundle/os-mt-ult.css

```diff
@@ -419,14 +419,14 @@ label.mt-feedback-consent-label {
     .mt-toc-container-ult {
         right: -17%;
         position: absolute !important;
-        top: 320px;
+        top: 270px;
     }
 
     .mt-toc-container-ult-fixed {
         margin: 15px 0px 15px 0px !important;
-        right: 6%;
+        right: 6.4%;
```

Comment:
This doesn't really work as it depends on the resolution - at least, the way the rest of the screen is built... Depending on the screen width, this can be less or more. :(

Reply:
If it's over 1400px, this is perfect. If less, it jumps. Hm.